doc/source/whatsnew/v0.25.0.rst

```diff
@@ -133,6 +133,7 @@ Other Enhancements
 - :meth:`DataFrame.describe` now formats integer percentiles without decimal point (:issue:`26660`)
 - Added support for reading SPSS .sav files using :func:`read_spss` (:issue:`26537`)
 - Added new option ``plotting.backend`` to be able to select a plotting backend different than the existing ``matplotlib`` one. Use ``pandas.set_option('plotting.backend', '<backend-module>')`` where ``<backend-module`` is a library implementing the pandas plotting API (:issue:`14130`)
+- :meth:`nargsort` handles ``ExtensionArray`` without calling ``np.asanyarray`` (:issue:`25439`)
```

Comment:
IIRC you had this different before, can you say that EA now argsorts Nans at the end

Reply:
Referencing :meth:`pandas.api.extensions.ExtensionArray.argsort`

Reply:
@makbigc can you update this?

Reply:
#26354 But the PR in which `ExtensionArray.argsort` places Nans at the end by `nargsort` hasn't been merged. Should we add the reference in the whatsnew entry?

Reply:
Ah got them mixed up. You can just remove this note then.

Reply:
removed.